pkg/util/log/annotation_scraper.go

```diff
@@ -36,16 +36,18 @@ import (
 type SelectorScraper struct {
 	client        kubernetes.Interface
 	namespace     string
+	integrationName string
 	labelSelector string
 	podScrapers   sync.Map
 	counter       uint64
 }
 
 // NewSelectorScraper creates a new SelectorScraper
-func NewSelectorScraper(client kubernetes.Interface, namespace string, labelSelector string) *SelectorScraper {
+func NewSelectorScraper(client kubernetes.Interface, namespace string, integrationName string, labelSelector string) *SelectorScraper {
```

Comment:
if possible I would not use the name integration in this package

Reply:
Ok, what's about `defaultContainerName` ?

Reply:
fine with me